packages/amplify-category-storage/src/provider-utils/awscloudformation/index.ts

```diff
@@ -164,3 +165,34 @@ const getHeadlessParams = (context: $TSContext) => {
     throw new Error(`Failed to parse storage headless parameters: ${err}`);
   }
 }
+
+export const console = async (amplifyMeta: $TSMeta, provider: string, service: string) => {
+  if (service === AmplifySupportedService.S3) {
+    const s3Resource = Object.values<any>(amplifyMeta[categoryName])
+      .filter((resource) => resource.service === service).pop();
+    if (!s3Resource) {
+      const errMessage = 'No S3 resources to open. You need to add a resource.';
+      printer.error(errMessage);
+      return;
+    }
+    const { BucketName: bucket, Region: region } = s3Resource.output;
+    const url = `https://s3.console.aws.amazon.com/s3/buckets/${bucket}?region=${region}`;
+    open(url, { wait: false });
+  } else if (service === AmplifySupportedService.DYNAMODB) {
+    type Pickchoice = { name: string, value: { tableName: string, region: string } };
+    const tables: Pickchoice[] = Object.values<any>(amplifyMeta[categoryName])
+      .filter((resource) => resource.service === service)
+      .map(resource => ({
+        name: resource.output.Name, 
+        value: { tableName: resource.output.Name, region: resource.output.Region }
+      }));
+    if (!tables.length) {
+      const errMessage = 'No DynamoDB tables to open. You need to add a resource.';
+      printer.error(errMessage);
+      return;
+    }
+    const { tableName, region } = await prompter.pick<'one', Pickchoice['value']>("Select DynamoDB table to open on your browser", tables);
+    const url = `https://${region}.console.aws.amazon.com/dynamodbv2/home?region=${region}#table?initialTagKey=&name=${tableName}&tab=overview`;
```

Comment:
is the `initialTagKey=&` query parameter needed?

Reply:
Its query parameter is not required, I will remove it.